tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAssetSetService.scala

```diff
@@ -0,0 +1,40 @@
+package com.linagora.tmail.james.jmap.publicAsset
+
+import jakarta.inject.{Inject, Named}
+import org.apache.commons.configuration2.Configuration
+import org.apache.james.jmap.api.model.{IdentityId, UploadId}
+import org.apache.james.jmap.core.AccountId
+import org.apache.james.jmap.mail.BlobId
+import org.apache.james.jmap.method.IdentityResolver
+import org.apache.james.mailbox.MailboxSession
+import reactor.core.scala.publisher.{SFlux, SMono}
+
+import scala.util.Try
+
+class PublicAssetSetService @Inject()(val identityResolver: IdentityResolver,
+                                      @Named("jmap") jmapConfiguration: Configuration) {
+
+  private val publicURIPrefix: String = jmapConfiguration.getString("publicAsset.publicURIPrefix", "")
```

Comment:
Does it make sense to give a service without a URI prefix?

IMO we can force a configuration here and fail if nothing is configured. Wou,d be cleaner than bad implicit values.